flake.nix

```diff
@@ -44,11 +44,13 @@
       in {
         devShells.default = pkgs.mkShell {
           name = "sg.nvim";
-          buildInputs = with pkgs; [
-            openssl
-            pkg-config
-            toolchain
-          ];
+          buildInputs = with pkgs;
+            [
```

Comment:
I am not sure what is the right style here. _I think it might be_
```nix
buildInputs = with pkgs; [
            openssl
            pkg-config
            toolchain
          ] ++ lib.optionals stdenv.isDarwin [darwin.apple_sdk.framework.Security];
```

Reply:
oooh. Missed that!

Reply:
This was autoformatted by something in my shell, so I think it's fine, right?

Reply:
it probably ran `nix fmt` so all good!